CHANGES.adoc

```diff
@@ -2,6 +2,10 @@
 
 This document describes the relevant changes between releases of the API model.
 
+== 0.0.99 Nov 16 2020
+
+- Add deletion add-on installation endpoint
```

Comment:
Can you do this in a separate commit, and also include `- Add Update method to addon installation resource`